scripts/create_ios_strings.js

```diff
@@ -43,12 +42,7 @@ function writeStringFile(plistStringJsonObj, lang, fileName) {
     fs.ensureDir(lProjPath, function (err) {
         if (!err) {
             var stringToWrite = jsonToDotStrings(plistStringJsonObj);
-            var buffer = iconv.encode(stringToWrite, 'utf8');
-
-            fs.open(lProjPath + '/' + fileName, 'w', function (err, fd) {
-                if (err) throw err;
-                fs.writeFileSync(fd, buffer);
-            });
+            fs.writeFileSync(path.join(lProjPath, fileName), stringToWrite);
```

Comment:
I was wondering exactly this! I didnt understand why there was encoding going on.... thanks!

Reply:
It seems that, before #35, the encoding used to write file was 'utf16' (requiring iconv-lite and buffer).
Back to using 'utf8' means we can simply use `writeFile` 👍 .